books/free-programming-books-langs.md

```diff
@@ -1996,6 +1997,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Why's (Poignant) Guide to Ruby](http://poignant.guide)
 
 
+#### Roda
+
+* [Mastering Roda](https://fiachetti.gitlab.io/mastering-roda) - Federico Iachetti, Avdi Grimm and Jeremy Evans
```

Comment:
Move this into Python section, more or less on line 1974. Lists alphabetization must be preserved.

Reply:
Well, why the Python section? Because roda is a ruby gem not a python library https://roda.jeremyevans.net

Reply:
Or you just think that is not enough to create a new section?

Reply:
Yes you are right, I fail with the section :weary:, but the line remains the same :wink: .

Reply:
I just know this book about roda, so I think that I will resubmit it adding to ruby section :)